diff --git a/lv_conf.defaults b/lv_conf.defaults
index db43103..6571bc4 100644
--- a/lv_conf.defaults
+++ b/lv_conf.defaults
@@ -1,8 +1,13 @@
-LV_COLOR_DEPTH	    32
-LV_MEM_SIZE	        (1024 * 1024)
-LV_USE_MATRIX       1
-LV_USE_FLOAT        1
-LV_USE_LOTTIE       1
+LV_COLOR_DEPTH	    16
+
+# LV_MEM_SIZE	        (1024 * 1024)
+LV_USE_STDLIB_MALLOC LV_STDLIB_CLIB
+LV_USE_STDLIB_STRING LV_STDLIB_CLIB
+LV_USE_STDLIB_SPRINTF LV_STDLIB_CLIB
+
+LV_USE_MATRIX       0
+LV_USE_FLOAT        0
+LV_USE_LOTTIE       0
 
 LV_USE_DRAW_SW_COMPLEX_GRADIENTS 1
 LV_OBJ_STYLE_CACHE      1
@@ -37,10 +42,11 @@ LV_FONT_UNSCII_8	1
 
 LV_USE_IMGFONT          1
 LV_USE_FS_STDIO         1
+LV_FS_DEFAULT_DRIVER_LETTER 'A'
 LV_FS_STDIO_LETTER      'A'
-LV_USE_THORVG_INTERNAL  1 
+LV_USE_THORVG_INTERNAL  0 
 LV_USE_LZ4_INTERNAL     1
-LV_USE_VECTOR_GRAPHIC   1
+LV_USE_VECTOR_GRAPHIC   0
 LV_USE_TINY_TTF         1
 LV_USE_BARCODE          1
 LV_USE_QRCODE           1
@@ -51,6 +57,9 @@ LV_USE_BMP              1
 LV_USE_LODEPNG          1
 
 LV_USE_LINUX_FBDEV              1
+LV_USE_WAYLAND              0
+
+LV_USE_EVDEV    1
 
 LV_USE_DEMO_WIDGETS         1
 LV_USE_DEMO_KEYPAD_AND_ENCODER 1
@@ -62,4 +71,7 @@ LV_USE_DEMO_FLEX_LAYOUT     1
 LV_USE_DEMO_MULTILANG       1
 LV_USE_DEMO_TRANSFORM       1
 LV_USE_DEMO_SCROLL          1
+
+LV_USE_DEMO_HIGH_RES        1
+LV_FONT_FMT_TXT_LARGE       1
  
diff --git a/lv_conf.h b/lv_conf.h
index d67c319..e080a4a 100644
--- a/lv_conf.h
+++ b/lv_conf.h
@@ -26,7 +26,7 @@
  *====================*/
 
 /** Color depth: 1 (I1), 8 (L8), 16 (RGB565), 24 (RGB888), 32 (XRGB8888) */
-#define LV_COLOR_DEPTH 32
+#define LV_COLOR_DEPTH 16
 
 /*=========================
    STDLIB WRAPPER SETTINGS
@@ -39,7 +39,7 @@
  * - LV_STDLIB_RTTHREAD:    RT-Thread implementation
  * - LV_STDLIB_CUSTOM:      Implement the functions externally
  */
-#define LV_USE_STDLIB_MALLOC    LV_STDLIB_BUILTIN
+#define LV_USE_STDLIB_MALLOC    LV_STDLIB_CLIB
 
 /** Possible values
  * - LV_STDLIB_BUILTIN:     LVGL's built in implementation
@@ -48,7 +48,7 @@
  * - LV_STDLIB_RTTHREAD:    RT-Thread implementation
  * - LV_STDLIB_CUSTOM:      Implement the functions externally
  */
-#define LV_USE_STDLIB_STRING    LV_STDLIB_BUILTIN
+#define LV_USE_STDLIB_STRING    LV_STDLIB_CLIB
 
 /** Possible values
  * - LV_STDLIB_BUILTIN:     LVGL's built in implementation
@@ -57,7 +57,7 @@
  * - LV_STDLIB_RTTHREAD:    RT-Thread implementation
  * - LV_STDLIB_CUSTOM:      Implement the functions externally
  */
-#define LV_USE_STDLIB_SPRINTF   LV_STDLIB_BUILTIN
+#define LV_USE_STDLIB_SPRINTF   LV_STDLIB_CLIB
 
 #define LV_STDINT_INCLUDE       <stdint.h>
 #define LV_STDDEF_INCLUDE       <stddef.h>
@@ -68,7 +68,7 @@
 
 #if LV_USE_STDLIB_MALLOC == LV_STDLIB_BUILTIN
     /** Size of memory available for `lv_malloc()` in bytes (>= 2kB) */
-    #define LV_MEM_SIZE (1024 * 1024)
+    #define LV_MEM_SIZE (64 * 1024U)          /**< [bytes] */
 
     /** Size of the memory expand for `lv_malloc()` in bytes */
     #define LV_MEM_POOL_EXPAND_SIZE 0
@@ -526,11 +526,11 @@
 #define LV_ATTRIBUTE_EXTERN_DATA
 
 /** Use `float` as `lv_value_precise_t` */
-#define LV_USE_FLOAT            1
+#define LV_USE_FLOAT            0
 
 /** Enable matrix support
  *  - Requires `LV_USE_FLOAT = 1` */
-#define LV_USE_MATRIX           1
+#define LV_USE_MATRIX           0
 
 /** Include `lvgl_private.h` in `lvgl.h` to access internal data and functions by default */
 #define LV_USE_PRIVATE_API		0
@@ -590,7 +590,7 @@
 /** Enable handling large font and/or fonts with a lot of characters.
  *  The limit depends on the font size, font face and bpp.
  *  A compiler error will be triggered if a font needs it. */
-#define LV_FONT_FMT_TXT_LARGE 0
+#define LV_FONT_FMT_TXT_LARGE 1
 
 /** Enables/disables support for compressed fonts. */
 #define LV_USE_FONT_COMPRESSED 0
@@ -710,7 +710,7 @@
 
 #define LV_USE_LIST       1
 
-#define LV_USE_LOTTIE     1
+#define LV_USE_LOTTIE     0
 
 #define LV_USE_MENU       1
 
@@ -790,7 +790,7 @@
 /* File system interfaces for common APIs */
 
 /** Setting a default driver letter allows skipping the driver prefix in filepaths. */
-#define LV_FS_DEFAULT_DRIVER_LETTER '\0'
+#define LV_FS_DEFAULT_DRIVER_LETTER 'A'
 
 /** API for fopen, fread, etc. */
 #define LV_USE_FS_STDIO 1
@@ -918,10 +918,10 @@
 
 /** Enable Vector Graphic APIs
  *  - Requires `LV_USE_MATRIX = 1` */
-#define LV_USE_VECTOR_GRAPHIC  1
+#define LV_USE_VECTOR_GRAPHIC  0
 
 /** Enable ThorVG (vector graphics library) from the src/libs folder */
-#define LV_USE_THORVG_INTERNAL 1 
+#define LV_USE_THORVG_INTERNAL 0 
 
 /** Enable ThorVG by assuming that its installed and linked to the project */
 #define LV_USE_THORVG_EXTERNAL 0
@@ -1174,7 +1174,7 @@
 #define LV_USE_TFT_ESPI         0
 
 /** Driver for evdev input devices */
-#define LV_USE_EVDEV    0
+#define LV_USE_EVDEV    1
 
 /** Driver for libinput input devices */
 #define LV_USE_LIBINPUT    0
@@ -1289,7 +1289,7 @@
 #endif
 
 /** High-resolution demo */
-#define LV_USE_DEMO_HIGH_RES        0
+#define LV_USE_DEMO_HIGH_RES        1
 
 /* Smart watch demo */
 #define LV_USE_DEMO_SMARTWATCH      0
diff --git a/main.c b/main.c
index c997aec..b082652 100644
--- a/main.c
+++ b/main.c
@@ -192,8 +192,9 @@ int main(int argc, char **argv)
     lv_linux_disp_init();
 
     /*Create a Demo*/
-    lv_demo_widgets();
-    lv_demo_widgets_start_slideshow();
+    // lv_demo_widgets();
+    // lv_demo_widgets_start_slideshow();
+    lv_demo_high_res_api_example("/root/lv_port_linux/lvgl/demos/high_res/assets", "/root/lv_port_linux/lvgl/demos/high_res/assets/img_lv_demo_high_res_lvgl_logo.png", NULL);
 
     lv_linux_run_loop();
 
